scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -214,8 +207,7 @@ def main():
                         'but it is missing.'
                         .format(SAMPLE_TEST_SUITE_THAT_IS_KNOWN_TO_EXIST))
 
-    if not (protractor_test_suites == travis_e2e_jobs and
-            travis_e2e_jobs == travis_e2e_scripts):
```

Comment:
Any specific reason for this change? Also, if we are going to have this then do we need to generate `travis_e2e_jobs`? If now, should we remove the code for that?

Reply:
There's still one check that checks formatting, so we can keep that. This test does test name to suite run correlation checks, which don't make sense any more given the test grouping.

Reply:
I mean `get_e2e_suite_names_from_jobs_travis_yml_file` doesn't make sense any more i.e, the job section doesn't have suite names, right? so should we change the method name or do we still need any check reacted to the job section?

Reply:
changed!